ext/ffi/ffi.c

```diff
@@ -4640,9 +4643,15 @@ static HashTable *zend_fake_get_gc(zend_object *ob, zval **table, int *n) /* {{{
 }
 /* }}} */
 
-static int zend_fake_cast_object(zend_object *obj, zval *result, int type)
+static int zend_cast_object(zend_object *obj, zval *result, int type)
```

Comment:
Why the rename?

Reply:
Because it isn't a *fake* cast anymore, is it?

Reply:
Still seems to be pretty fake to me ^^ `zend_cast_object` sounds like how the standard object handler would be called. (I guess the problem here is more that all of this is prefixed by zend_ for some reason, even though it has nothing to do with ZE.)

Reply:
Oh, seen this way, the renaming was a bad idea. Thanks!